qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/ls/commons/snippets/Snippet.java

```diff
@@ -95,6 +97,14 @@ public void setSortText(String sortText) {
 		this.sortText = sortText;
 	}
 
+	public String getParent() {

```

Comment:
Please dont update classes from commons with qute specific feature.
For managing specific data like parent you must implemt ISnippetContext and déclare in json à context which have à parent. See sample with lsp4mp with context/type